lab2/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.myapplication">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <receiver
+            android:name=".MyReceiver"
+            android:enabled="true"
+            android:exported="true"></receiver>
```

Comment:
Security warning: do not export your components unless they really need to be accessed from other apps. This is also a security practice mentioned in the lab: https://ocw.cs.pub.ro/courses/smd/laboratoare/03

> The Android components can have the exported argument declared in the manifest. If true, it allows other apps to access it. In the case of a service, make sure it is false (default value if no intent filters are declared) if you plan to use it only inside your app. See here more details about this and a complete list of service parameters.